### PR TITLE
Uifix button colors 1177

### LINKED ISSUE #1177: [evidence] Upload file button - must have sufficient color contrast
## Summary

> Please ensures that the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

## Design Detail

Screen Shot 2020-01-19 at 7 23 23 PM

## Recommended Solution

To solve this violation, you need to fix the following:
Element has insufficient color contrast of 4.02 (foreground color: #ffffff, background color: #3182ce, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1

Screen Shot 2020-01-19 at 7 27 25 PM

## Checker result 
Screen Shot 2020-01-19 at 7 37 36 PM

### PR DESCRIPTION
**Pull request by amazingphilippe**
# Description

Passing button color contrast. 

Fixes #1177 

- [ ] Bug fix

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made any needed changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

# Required followup work

The color pairs could be automatically generated using a library like chroma.js

### CODE REVIEW

File: f2/src/forms/LocationInfoForm.js [hunk 1]
```diff
@@ -39,7 +36,7 @@ export const LocationInfoForm = props => {
                   </Trans>
                 </FormHelperText>
                 <TextInput
-                  id="email"
+                  id="postalCode"
```
Comment (sastels) [inline f2/src/forms/LocationInfoForm.js]: I think you might have branched this off your other PR branch rather than master.

**Comment by sastels** [comment]
This is also passing WCAG AA but failing AAA. Should we darken up the buttons more?

**Comment by amazingphilippe** [comment]
Font size for links and buttons is considered large at over 18 points, therefore can pass AAA!

File: f2/src/forms/LocationInfoForm.js (hunk 1 above)
Comment (amazingphilippe) [reply f2/src/forms/LocationInfoForm.js]: Gone!

**Pull request merged**